article/test/AnalyticsFeatureTest.scala

```diff
@@ -20,26 +20,7 @@ import conf.Configuration
 
     // Scenarios
 
-    scenario("Omniture tracks user actions") {
-
-      Given("I am on an article entitled 'Olympic opening ceremony will recreate countryside with real animals'")
-      goTo("/sport/2012/jun/12/london-2012-olympic-opening-ceremony") { browser =>
-        import browser._
-
-        Then("the Omniture webbug should record my visit")
-        val webbug = findFirst("#omnitureNoScript img")
```

Comment:
For reference, answer to this:

"This noscript Omniture tag no longer exists for new Omniture implementations so the only reason we have it as part of dotcoms omniture implementation is down to how long ago we plugged it in.

The main issue with noscript tags we've had in the past is they only parse in a vert vanilla page view - meaning you then have a page view recorded that doesn't have the full range of custom dimensions."